benchmarks/common/benchmark_runner.hpp

```diff
@@ -303,22 +304,47 @@ struct BenchmarkRunner {
   ~BenchmarkRunner() {
     benchmark::Shutdown();
   }
-
+    
 private:
   static void run_benchmark(benchmark::State& state, const Options& options,  Gemm gemm_op) {
-    state.counters["runtime_ms"] = 0;
+    auto tflop = ((2.0 * options.m * options.n * options.k * options.l) * 1e-12);
+    auto giga_bytes_transferred = ((options.m * options.k) * sizeof(ElementA) +
+                                    (options.k * options.n) * sizeof(ElementB) +
+                                2 * (options.m * options.n) * sizeof(ElementC)) * 1e-9;
```

Comment:
Do we want to add the `C` bytes only if `beta != 0`? e.g. something like
```suggestion
                                (options.beta != 0 ? 2 : 1) * (options.m * options.n) * sizeof(ElementC)) * 1e-9;
```